deployment/paiLibrary/clusterObjectModel/paiObjectModel.py

```diff
@@ -311,13 +312,26 @@ def getPaiWebPortalUri(self):
     def getK8sApiServerUri(self):
 
         ip = self.rawData["kubernetesConfiguration"]["kubernetes"]["load-balance-ip"]
-        ret = "http://{0}:{1}".format(ip, 8080)
+        ret = ""
+        if self.rawData["kubernetesConfiguration"]["kubernetes"]["k8s-type"] == "aks":
+            ret = "https://{0}:{1}".format(ip, 443)
+        else:
```

Comment:
Actually this section is used to determin the K8S API server port. On-prem it configured as 8080, on AKS it configured as 443. So I suggest we can add a kubernetes/api-server-port config item and configure it as 443 instead of adding a so called "k8s-type" as "aks". This should be more general and direct.

Reply:
move it as api-server-port config and remove k8s-type config. for 8080 to 443, as serveral files have dependecy and use 8080 hard code config, need confirm with yundong, currently just config 8080 as a temp and work config

Reply:
pai use http, aks use https, also make it as a config.